docs/asciidoc/static/configuration.asciidoc

```diff
@@ -358,7 +358,7 @@ What's an expression? Comparison tests, boolean logic, and so on!
 You can use the following comparison operators:
 
 * equality: ==,  !=,  <,  >,  <=,  >=
-* regexp: =~, !~
+* regexp: =\~, !~
```

Comment:
No, and this is one annoyance with asciidoc. :) See thread here: https://groups.google.com/forum/#!topic/asciidoc/UpzkNXrxtnQ